src/FixedDataCellGroupLegacyFunction.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import FixedDataTableTranslateDOMPosition from './FixedDataTableTranslateDOMPosition';
+import cx from './vendor_upstream/stubs/cx';
+function CellGroupLegacy(props) {
+  var style1 = {
```

Comment:
Here and every other place, Please avoid using name like `style1`, `style2` instead use meaningful and descriptive names for variables for better code readability.

Reply:
Done